### PR TITLE
Allow to add plugins manually

### PR DESCRIPTION
**Pull request by shargon**
With these changes it's easier for me to debug and test a plugin.

I just need to add all the projects together and add a temporary line to add this plugin.

image

### CODE REVIEW

**Comment by erikzhang** [comment]
If you load a plugin after `NeoSystem` is created, the `OnPluginsLoaded()` method of this plugin will never be called.

File: neo/Plugins/Plugin.cs [hunk 1]
```diff
@@ -91,28 +91,32 @@ protected IConfigurationSection GetConfiguration()
             return new ConfigurationBuilder().AddJsonFile(ConfigFile, optional: true).Build().GetSection("PluginConfiguration");
         }
 
+        public static void LoadPlugin(Assembly assembly)
```
Comment (erikzhang) [inline neo/Plugins/Plugin.cs]:
```c#
public static T LoadPlugin<T>() where T : class
```

Comment (shargon) [reply neo/Plugins/Plugin.cs]: cool!

**Comment by shargon** [comment]
If i create a instance is automatially loaded, so I don't need it 🤣 

I just need to do : `new NeoStatsPlugin.StatsPlugin();` and is loaded :P

**Pull request closed without merging**